src/lib/services/ifc-export-service.ts

```
@@ -0,0 +1,563 @@
+            );
+          }
+
+          window.pyodide = pyodide;
+          resolve(pyodide);
+        }
+      } catch (error) {
+        console.error("Error during Pyodide initialization:", error);
+        pyodidePromise = null;
+        reject(error);
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)